### DISCUSSION TITLE
Roadmap for 5.0 Release

### DISCUSSION THREAD

**Discussion by cptkng23** [vladimir-bukhtoyarov/bucket4j#152] [General]
Hi Vladimir.

I'm very interested in some of the 5.0 features, such as your described here: https://gitter.im/vladimir-bukhtoyarov/bucket4j?at=5fb5013b8d286f2076b844e4

Do you have a timeline for the 5.0 release. Or, if those features make it into 7.0, for that 7.0 release.

Thanks for this awesome lib!

**Comment by vladimir-bukhtoyarov** [comment]
Hello @cptkng23 

```5.0``` has been transformed to ```7.0``` because of reasons described in this comment https://github.com/vladimir-bukhtoyarov/bucket4j/pull/142#issuecomment-752377882

According to release plans for ```7.0```: there is no clear release date. But I am going to resume my efforts related to documentation in march 2021. In my expectations, it will take 1-2 months. Currently, you can use release ```5.0.0-beta.01-09-2020``` (without documentation).

**Reply by cptkng23** [reply]
Thanks for your reply, @vladimir-bukhtoyarov . That sounds good.
